orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/artifacts/FindArtifactFromExecutionTask.java

```diff
@@ -64,11 +64,12 @@ public TaskResult execute(@Nonnull Stage stage) {
           artifactUtils.getArtifactsForPipelineId(pipeline, executionOptions.toCriteria());
     }
 
-    Set<Artifact> matchingArtifacts =
-        artifactUtils.resolveExpectedArtifacts(expectedArtifacts, priorArtifacts, null, false);
+    ArtifactResolver.ResolveResult resolveResult =
+        ArtifactResolver.getInstance(priorArtifacts, null, false)
```

Comment:
more nitpicky stuff, but I think it's helpful to add a parameter comment to constant parameters... go/java-practices/comments#arguments

```
ArtifactResolver.getInstance(
    priorArtifacts, /* priorArtifacts= */ null, /* requireUniqueMatches= */ false);
```

That way readers can see what's going on without having to click through to `#getInstance`. You can add these comments in IntelliJ with `ctrl-alt-/` (this is an internal extension, doesn't work outside Google)

Reply:
Interesting...I've seen these around before but haven't been in the habit of adding them.  The way IntelliJ is configured for me, the parameter name shows up in the UI:
![image](https://user-images.githubusercontent.com/1391982/72835127-23955800-3c58-11ea-9638-4b26fcad9dbd.png)
But I can see the advantage of having this directly in the text and not depend on a particular IDE.  I can't seem to get that shortcut to work, but maybe I don't have the right extension installed...I'll see if I can get it to work.

Reply:
Ok, couldn't get the extension to work but did use search/replace to add these!